{"uid":"e0487065f72ceab6","name":"test_create_user_with_invalid_phone_number_returns_400","fullName":"test.users.test_post_user#test_create_user_with_invalid_phone_number_returns_400","historyId":"df1c10ea0e47f1b097e69a146ad0b075","time":{"start":1726087512930,"stop":1726087513075,"duration":145},"status":"failed","statusMessage":"AssertionError: Expected status code 400, but got 403. Response text:","statusTrace":"@pytest.mark.createuser\n    def test_create_user_with_invalid_phone_number_returns_400():\n        url = EndpointUser.get_base_user()\n        data = create_user_data(phone_number=\"+59173\")\n        response = UserService.create_user(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/users/test_post_user.py:182: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [403]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 403. Response text:\n\ncore/assertions/status_code_assertion.py:37: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1726087466101,"stop":1726087466118,"duration":17},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Expected status code 400, but got 403. Response text:","statusTrace":"@pytest.mark.createuser\n    def test_create_user_with_invalid_phone_number_returns_400():\n        url = EndpointUser.get_base_user()\n        data = create_user_data(phone_number=\"+59173\")\n        response = UserService.create_user(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/users/test_post_user.py:182: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [403]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 403. Response text:\n\ncore/assertions/status_code_assertion.py:37: AssertionError","steps":[],"attachments":[{"uid":"abe9d526eed2068b","name":"log","source":"abe9d526eed2068b.txt","type":"text/plain","size":1520},{"uid":"f5c32ab0c6dad129","name":"stderr","source":"f5c32ab0c6dad129.txt","type":"text/plain","size":1589}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true},"afterStages":[],"labels":[{"name":"tag","value":"createuser"},{"name":"parentSuite","value":"test.users"},{"name":"suite","value":"test_post_user"},{"name":"host","value":"fv-az1435-834"},{"name":"thread","value":"2084-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.users.test_post_user"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["createuser"]},"source":"e0487065f72ceab6.json","parameterValues":[]}